srcpkgs/Converseen/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'Converseen'
+pkgname=Converseen
+version=0.9.8.1
+revision=1
+archs="i686* x86_64*"
```

Comment:
now with cmake style will compile for all archs when added some packages to hostmakedepends, see build log